### DISCUSSION TITLE
Video match text regex capture groups

### DISCUSSION THREAD

**Discussion by rushsteve1** [WarmUpTill/SceneSwitcher#897] [Q&A]
Hello! I am trying to use the `Video > Match Text` condition with a Regular Expression for the string, and I was wondering if there is a way to get the capture groups from that regular expression as variables that can be used in Actions?

For example I want to a game's Round Number from the screen. So my regex would be `Round (\d+)` where I want to capture the number matched by `\d+`.

**Comment by WarmUpTill** [comment]
Yes that is possible using a setup similar to this:

![grafik](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/241dc35c-f746-4584-bb62-fce1a2e00024)

You first assign the text which the video condition detected to a variable and then use the same regular expression again to extract the x-th capture group.
